packages/plotkicadsch/plotkicadsch.0.1.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "Jean-Noel Avila <jn.avila@free.fr>"
+author: "Jean-Noel Avila <jn.avila@free.fr>"
+homepage: "https://jnavila.github.io/plotkicadsch/"
+bug-reports: "https://github.com/jnavila/plotkicadsch/issues"
+license: "ISC"
+dev-repo: "https://github.com/jnavila/plotkicadsch.git"
+build: [
+  [ "jbuilder" "subst"] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+depends: [
+ "ocamlfind"
```

Comment:
If you don't link your program with `Findlib` libraries, `ocamlfind` can be just a `{build}` dependency. You seem to also need `jbuilder` as a dependency, also only a `{build}` dependency.